handler/src/main/java/io/netty/handler/ssl/OpenSsl.java

```diff
@@ -381,7 +393,32 @@
             TLSV13_SUPPORTED = false;
             IS_BORINGSSL = false;
             EXTRA_SUPPORTED_TLS_1_3_CIPHERS = EmptyArrays.EMPTY_STRINGS;
+            EXTRA_SUPPORTED_TLS_1_3_CIPHERS_STRING = StringUtil.EMPTY_STRING;
+        }
+    }
+
+    static String checkTls13Ciphers(InternalLogger logger, String ciphers) {
+        if (IS_BORINGSSL && !ciphers.isEmpty()) {
+            assert EXTRA_SUPPORTED_TLS_1_3_CIPHERS.length > 0;
+            Set<String> boringsslTlsv13Ciphers = new HashSet<String>();
+            Collections.addAll(boringsslTlsv13Ciphers, EXTRA_SUPPORTED_TLS_1_3_CIPHERS);
+            boolean ciphersNotMatch = false;
+            for (String cipher: ciphers.split(",")) {
+                if (boringsslTlsv13Ciphers.isEmpty()) {
+                    ciphersNotMatch = true;
+                    break;
+                }
+                boringsslTlsv13Ciphers.remove(cipher);
+            }
+            if (ciphersNotMatch || !boringsslTlsv13Ciphers.isEmpty()) {
+                logger.info(
+                        "BoringSSL doesn't allow to enable or disable TLSv1.3 ciphers explicitly." +
+                                " The default TLSv1.3 ciphers will be used: '{}'.",
+                        Arrays.toString(EXTRA_SUPPORTED_TLS_1_3_CIPHERS));
```

Comment:
1. Consider using `EXTRA_SUPPORTED_TLS_1_3_CIPHERS_STRING` to avoid array -> to string conversion.
2. Consider also printing what `ciphers` users originally provided. It will help them to understand how they are different.